qualification/report/katdoc.sty

```diff
@@ -103,147 +123,121 @@
           \hline
         \end{tabularx}
       \end{table}
+    \end{center}
+  \end{titlepage}
+  %\stepcounter{page}
+}
 
-      \newpage{}
+\newcommand{\makeInfoPage}{%
+  \begin{center}
+    % lots of padding for signatures
+    \renewcommand{\arraystretch}{1.5}
 
-      % lots of padding for signatures
-      \renewcommand{\arraystretch}{2.0}
+    % Approval Table
+    \LARGE
+    \textbf{Document Approval}
 
-      % Approval Table
-      \LARGE
-      \textbf{Document Approval}
+    \normalsize
+    \begin{tabularx}{\textwidth}{|l|l|l|l|X|X|}
+      \hline
+      \textbf{Function} & \textbf{Name} & \textbf{Designation} & \textbf{Affiliation} & \textbf{Date} & \textbf{Signature} \\
+      \hline
+      \docApproval%
```

Comment:
Some lines end with `%` characters, any particular reason(s) why that is? I thought earlier on it meant "it's a multiline command coming", perhaps e.g. `\docApproval` is a multiline cmd declared elsewhere?

EDIT: Same with `docHistory` and `docSoftware` further down, previously they didn't have a trailing `%`.

Reply:
There's a tool called checklatex (I think) which moans about there being spaces right after the command.

There are two ways to silence that which are technically more correct, either put a % directly after, or an empty {} I think. I usually default to the former.

Reply:
It seems a bit odd that it would complain about spaces after a command, because a "feature" of commands is that they eat the following whitespace. Normally when I've seen end-of-line `%` in LaTeX it's to prevent the end-of-line being interpreted as whitespace that ends up as space in the document e.g. `\newcommand{\makeInfoPage}{%`

Reply:
Without it, `chktex` (that's what it's called) gives me this:
![image](https://user-images.githubusercontent.com/10864691/232044706-dcc86ab7-b62d-4941-9786-1293e7245dbd.png)

With the `%`, the message goes away.

I can remove them if you want, and just disable the warning. They don't actually affect the output. I just got the impression that it might be considered best practice, since chktex complains.